spi/identity-hub-spi/build.gradle.kts

```diff
@@ -0,0 +1,11 @@
+plugins {
+    `java-library`
+}
+
+val jetBrainsAnnotationsVersion: String by project
+val swagger: String by project
+
+dependencies {
+    api("org.jetbrains:annotations:${jetBrainsAnnotationsVersion}")
+    implementation("io.swagger.core.v3:swagger-jaxrs2-jakarta:${swagger}")
```

Comment:
Why does the SPI rely on Swagger?

Reply:
This is needed because it contains the model, we need to use the @Schema annotation to specify a good description used when generating the open api yaml file.

Reply:
It looks like parts of this module then are not an SPI module but an API module. We should have a distinction between classes that a client would use (API) and classes that would be implemented by a service (SPI).

Reply:
I could move these classes to an identity-hub-dtos module, what do you think?
I would just keep the VerifiableCredentials in spi then.

Reply:
I moved them to a dtos module.